src/current/v23.1/role-based-audit-logging.md

```diff
@@ -0,0 +1,151 @@
+{{site.data.alerts.end}}
+
+## Configure Logging
+
+### Prerequisites
+
+You must have:
+
+- [`admin` role]({% link {{ page.version.version }}/security-reference/authorization.md %}#admin-role) or `MODIFYCLUSTERSETTING` [system privilege]({% link {{ page.version.version }}/security-reference/authorization.md %}#supported-privileges) to configure the cluster setting,
+- an [Enterprise license configured]({% link {{ page.version.version }}/licensing-faqs.md %}#set-a-license), and
+- your [logging infrastructure configured]({% link {{ page.version.version }}/configure-logs.md %}), particularly the [location of audit logs](#file-storage-location).
+
+### Syntax of audit settings
+
+With the [`sql.log.user_audit` cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}#setting-sql-log-user-audit), you can set an audit logging configuration using a table-like syntax, as seen in the [Examples](#examples). Each row in the configuration represents an audit setting and must be separated by a new line. An audit setting is comprised of two columns separated by a space:
+
+- `USER/ROLE`: the name of the user or role you want to audit.
+    - The *special* `USER/ROLE` value `ALL` applies the audit setting to *all* users.
+- `STATEMENT_FILTER`: the statement filter for the role. The following keywords are valid:
+    - `ALL` will enable audit logging for all SQL statements for the given audit setting.
+    - `NONE` will exclude audit logging for all SQL statements for the given audit setting.
+
+Once `sql.log.user_audit` is set, the default behavior is for role-based SQL audit logging to take effect immediately within user sessions.
+
+### Matching order
+
+{{site.data.alerts.callout_info}}
+The order in which the audit settings in the configuration are specified is important, as seen in the [example of a user with multiple roles](#user-with-multiple-roles). When determining whether to emit a log, the first audit setting that matches a user is the audit setting that gets applied. 
+{{site.data.alerts.end}}
+
+For each statement executed, CockroachDB gets the user's role memberships and iterates through the audit settings, looking for a matching audit setting by role or username. At the first audit setting match, CockroachDB stops iteration. If auditing is enabled -  - the `STATEMENT_FILTER` is set to `ALL` - for the matching audit setting, the statement is logged as an audit event. If the `STATEMENT_FILTER` is set to `NONE`, the statement is not logged.
+
+### Validate setting
+
+You can check the value of the `sql.log.user_audit` setting by running the [`SHOW CLUSTER`]({% link {{ page.version.version }}/show-cluster-setting.md %}) command:
```

Comment:
Should be `SHOW CLUSTER SETTING` instead of `SHOW CLUSTER`

Reply:
updated